custom/aaa/models.py

```diff
@@ -142,18 +142,26 @@ def agg_from_person_case_ucr(cls, domain, window_start, window_end):
         return """
         INSERT INTO "{woman_tablename}" AS child (
             domain, household_case_id, person_case_id, opened_on, closed_on,
-            dob, marital_status, sex, migration_status
+            dob, marital_status, sex, migration_status, age_marriage,
+            has_aadhar_number, husband_name, contact_phone_number,
```

Comment:
I'm going to keep this because the app spells it `aadhar_number`. I think by using the same slugs between the app and the models it makes it more greppable, though I agree if the app did not already have it I would change it to the approved spelling